packages/realm/src/schema/from-binding.ts

```diff
@@ -36,7 +36,7 @@ const TYPE_MAPPINGS: Record<binding.PropertyType, PropertyTypeName | null> = {
   [PropertyType.Mixed]: "mixed",
   [PropertyType.ObjectId]: "objectId",
   [PropertyType.Decimal]: "decimal128",
-  [PropertyType.UUID]: "uuid",
+  [PropertyType.Uuid]: "uuid",
```

Comment:
This makes sense to me but it is worth noting that the BSON library refers to `UUID` as `UUID` and provides both `ObjectId` and `ObjectID` as aliases. Not terribly important but seems a little counterintuitive to diverge from it in our internal naming.

Reply:
Good point 👍  Yeah I'd be interested to hear what the team thinks :)

Reply:
@realm/realm-js-team good opportunity to test GitHub teams

Reply:
This is an enumerator, not the type. I don't think it is a good idea to have one-of exceptions like that in code generated from the spec. Either we should say we want to use a rule to convert all names to a standard style, or we should just use the names as they are defined in the spec.